File: src/SOP8/Base/History/Base.History.IBLL/Response/ResponseSOPComponentHistory.cs
using System.Collections.Generic;
using Response;

namespace Base.History.IBLL.Response
{
    using Models.SOP;

    public class ResponseSOPComponentHistory : MessageResult
    {
        private List<SOPHistoryComponentData> m_sopComponentHistoryDatas = new List<SOPHistoryComponentData>();
        public List<SOPHistoryComponentData> SopComponentHistoryDatas
        {
            get { return m_sopComponentHistoryDatas; }
            set { m_sopComponentHistoryDatas = value; }
        }

        public ResponseSOPComponentHistory()
            : base()
        {
        }

        public ResponseSOPComponentHistory(bool success, string message)
            : base(success, message)
        {
        }
    }
}

File: src/SOP8/Egovframework/Base/base-history-ibll/src/main/java/egovframework/base/history/ibll/response/ResponseSOPComponentHistory.java
package egovframework.base.history.ibll.response;

import egovframework.base.history.ibll.models.sop.SOPHistoryComponentData;
import egovframework.base.response.MessageResult;

import java.util.ArrayList;
import java.util.List;

public class ResponseSOPComponentHistory extends MessageResult {

    private List<SOPHistoryComponentData> sopComponentHistoryDatas = new ArrayList<>();

    // Constructors
    public ResponseSOPComponentHistory() {
        super();
    }

    public ResponseSOPComponentHistory(boolean success, String message) {
        super(success, message);
    }

    // Getter and Setter
    public List<SOPHistoryComponentData> getSopComponentHistoryDatas() {
        return sopComponentHistoryDatas;
    }

    public void setSopComponentHistoryDatas(List<SOPHistoryComponentData> sopComponentHistoryDatas) {
        this.sopComponentHistoryDatas = sopComponentHistoryDatas;
    }
}